registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/workflow/dto/SearchFilter.java

```diff
@@ -0,0 +1,34 @@
+package io.mosip.registration.processor.core.workflow.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Dtp to hold the search parameters
+ * 
+ * @author Abhishek Kumar
+ * @since 1.0.0
+ */
+
+@Data
+@AllArgsConstructor()
+@NoArgsConstructor
+public class SearchFilter {
+	private String value;
+
+	// @Pattern(regexp = "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}Z$",
+	// message = "Invalid date time pattern")
+	private String fromValue;
+
+	// @Pattern(regexp = "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}Z$",
+	// message = "Invalid date time pattern")
+	private String toValue;
+
+	// @NotBlank
+	private String columnName;
+
+	// @NotNull
+	private String type;
```

Comment:
What all values are possible in this type ?

Reply:
value type eg:string

Reply:
string is one type, what are all the types, can we list and also I see we are not doing anything with this value. My understanding is this should be type like equals, between etc, so based on this we will take a decision to use the direct value or use from and to values

Reply:
we r not supporting for type only column name and value